src/actions/tabs.action-creator.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+import { SWITCH_TAB, } from './types';
+
+export function switchTabActionCreator(selectedTab){
```

Comment:
Cool. Yeah don't change anything just yet. If you can gimme a demo and walk through of this before we merge I'll be able to better understand where you're coming from and add a bit color from there.